packages/compass-serverstats/src/components/index.tsx

```diff
@@ -86,6 +78,31 @@ function PerformancePanel({
   );
 }
 
+function PerformancePanelMsgs() {
+  const [, forceUpdate] = useState({});
+
+  useEffect(() => {
+    TopStore.listen(() => {
+      forceUpdate({}); // Trigger the component refresh when the store is updated.
+    }, this);
+  }, []);
+
+  return (
+    <div>
+      {ServerStatsStore.isMongos && (
```

Comment:
Just to align the patterns I wonder if it would make sense to add `ServerStatsStore.listen` in the same effect where you subscribe to `TopStore` (otherwise it might be confusing why we are listening to one store but not the other) even if the value doesn't change in runtime